extensions/amp-instagram/0.1/amp-instagram.js

```diff
@@ -61,7 +61,10 @@ class AmpInstagram extends AMP.BaseElement {
      * @private {?Promise}
      */
     this.iframePromise_ = null;
-
+    /**
+     * @private {?Element}
+     */
+    this.placeholderWrapper_ = null;
```

Comment:
We can make this an actual placholder, then take advantage of `#togglePlaceholder` to hide and show it.